charts/minio/1.3.11/questions.yaml

```diff
@@ -137,7 +137,7 @@ questions:
             type: int
             min: 9000
             max: 65535
-            default: 9002
+            default: 9001
```

Comment:
This conflicts with nextcloud app from official catalog, let's please keep `9002` as the default value